R/spinMyR.R

```diff
@@ -213,8 +215,14 @@ spinMyR <- function(file, wd, outDir, figDir, outSuffix,
               fileMd,
               quiet = !verbose,
               envir = spinMyR_Env)
+  if (!keepRmd) {
+    unlink(fileRmd)
```

Comment:
You're right, I just blindly copied the unlink code from elsewhere in the
code, but that was for directories not for files. Disregard my comment :)

---
http://deanattali.com

On 23 October 2015 at 00:13, Kirill Müller <notifications@github.com> wrote:

> In R/spinMyR.R
> <https://github.com/daattali/rsalad/pull/15#discussion_r42838444>:
>
> > @@ -213,8 +215,14 @@ spinMyR <- function(file, wd, outDir, figDir, outSuffix,
> >                fileMd,
> >                quiet = !verbose,
> >                envir = spinMyR_Env)
> > +  if (!keepRmd) {
> > +    unlink(fileRmd)
>
> I'm not sure I need recursive, and why would we mute the warning when we
> can be reasonably sure that the file exists (otherwise knit would have
> thrown an error already)?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/daattali/rsalad/pull/15/files#r42838444>.
>